### PR TITLE
Improving ACME-related docs

### PR DESCRIPTION
**Pull request by ser**

### CODE REVIEW

File: docs/acme-integration.rst [hunk 1]
```diff
@@ -153,6 +153,33 @@ configuration active, it will check for validity of the signed certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
+I want a certifcate for subdomains but domain
```
Comment (drybjed) [inline docs/acme-integration.rst]: "but not the apex domain"

File: docs/acme-integration.rst [hunk 2]
```diff
@@ -153,6 +153,33 @@ configuration active, it will check for validity of the signed certificate, and
...
+---------------------------------------------
+
+Yes, it's possible :-) Please consult the example and create your own similar
+configuration. In the example we create a certificate for ``logs.it-zone.org``
+and ``mon.it-zone.org`` subdoimains, without creating cert for ``it-zone.org``
```
Comment (drybjed) [inline docs/acme-integration.rst]: Perhaps use the `example.com` or `example.org` domain instead of a custom domain. These are reserved for documentation.

**Comment by ypid** [comment]
Hey guys. I just rebased ontop of this PR and reworked it a bit as well in #81. Is [this](https://github.com/debops/ansible-pki/pull/81/commits/1a2c6865b8dfcfa415a3d60897318a2847faa256) ok with you?

**Comment by drybjed** [comment]
Sounds good to me, although I would revise the note about the realm name as commented.

**Pull request merged**